Dalamud.Boot/xivfixes.cpp

```diff
@@ -556,6 +556,37 @@ void xivfixes::clr_failfast_hijack(bool bApply)
     }
 }
 
+void xivfixes::keep_unhandled_exception_filter(bool bApply)
```

Comment:
Could you add a comment explaining what this does? I assume it's to prevent the game from setting its exception filter?

Reply:
Yeah, will add a comment